server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -818,10 +833,6 @@ private void doStop() {
         this.pm = null;
         this.repositoryWorker = null;
         this.sessionManager = null;
-        if (meterRegistry != null) {
-            meterRegistry.close();
```

Comment:
As of now, don't we close `meterRegistry` any more?

Reply:
I thought the lifecycle of the `MeterRegistry` is now independent from the `CentralDogma` server.

For instance, I was thinking if `Flags.meterRegistry` is used in `Application A` and `CentralDogma`, should `Flags.meterRegistry` be closed when `CentralDogma` is stopped although `ApplicationA` is running?

Reply:
> MeterRegistry is now independent from the CentralDogma server since it is configurable from CentralDogmaBuilder.

It may be right or not. It is still possible for CentralDogma server to create a `PrometheusMeterRegistry` internally which should go along with the server.

Reply:
Ideally, I think the best way would be to only remove all metrics that are related to `centraldogma`.
I think this would involve 
1) adding a prefix like `centraldogma.*` to all metrics
2) removing only metrics with the prefix `centraldogma.*`

Having said this, this is probably a breaking change and the scope of this PR would become a lot bigger.

> It is still possible for CentralDogma server to create a PrometheusMeterRegistry internally which should go along with the server.

I think it is possible to store the instantiated `PrometheusMeterRegistry` and remove it when closing the `CentralDogma` server. I think this will only remove the child registry and the meters will still be attached to the `CompositeMeterRegistry`.

Reply:
[6086a8f](https://github.com/line/centraldogma/pull/773/commits/6086a8f4d1fa9a7a10978a48af61cf1bba360a08) added a commit for this.

Let me know if this is what you meant 🙏

Reply:
Looks good! 🙇‍♂️